sn_node/src/replication.rs

```diff
@@ -136,3 +138,32 @@ impl Node {
         });
     }
 }
+
+pub struct ReplicationManager {
+    max_concurrent_replications: usize,
+    active_replications: Arc<AtomicUsize>,
+}
+
+impl ReplicationManager {
```

Comment:
currently, there is `ReplicationFetcher` which limits the `concurrent fetch` to be `MAX_PARALLEL_FETCH` (20).

This is somehow duplicated ?

Reply:
Looks like dead code, it's not used anywhere. No clue why this is in this PR.